app/server.py

```diff
@@ -209,9 +209,33 @@ def delete_prod_info(prod_id):
 @app.route('/inventory/<int:prod_id>', methods=[PUT])
 def update_prod_info(prod_id):
     """
-    Update prodcut information
+    Update ProdcutInformation
 
-    This endpoint will update a ProductInformation based on data that id posted in the body
+    This endpoint will update product inventory information (by id) based on data posted in the body
+    ---
+    tags:
+        -   Inventory
+    consumes:
+        -   application/json
+    produces:
+        -   application/json
+    parameters:
+        -   name: prod_id
+            in: path
+            description: ID of product information to be updated
+            type: integer
+            required: true
+        -   in: body
+            name: body
+            schema:
+                id: data
```

Comment:
add a `ref` to Products definition under schema

Reply:
addressed